add received transaction history

jkbktl opened the pull request:
based on #703 

to-do
- ✅ add links to explorer
- ✅ add exchange rate to EUR
- ✅ fix images for avatars of the tokens and NFTs
- ✅ fix sorting & merging outgoing/incoming transactions

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 0c231d4e | [#1](https://ci.status.im/job/status-web/job/prs/job/linux/job/wallet/job/PR-722/1/) | 2025-06-25 12:55:50 | ~1 min | `wallet` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Extension-250625-125447-0c231d-pr722-chrome.zip) |

jkbktl commented:
@jinhojang6 update:

- added exchange rate
- fixed token images (if not in our token list than show fallback with first four letter of symbol)
- formatted token amount and token value

<img width="997" alt="Screenshot 2025-06-30 at 8 32 40" src="https://github.com/user-attachments/assets/3b122682-1284-4d73-ba46-ced158600293" />

jkbktl commented on packages/wallet/src/components/activity-list/components/activity-item.tsx:
```diff
@@ -0,0 +1,128 @@
+      </div>
+      <div className="flex items-center justify-end text-13 font-400 text-neutral-50">
+        <a
+          href={`https://etherscan.io/tx/${activity.hash}`}
```
good enough for mvp-0

jkbktl commented on apps/wallet/src/components/action-buttons.tsx:
```diff
@@ -22,7 +22,6 @@ import { TabLink } from './tab-link'
 
 type Props = {
-  address: string
   pathname: string
```
For future use.

jkbktl commented on apps/wallet/src/hooks/use-activities.tsx:
```diff
@@ -0,0 +1,99 @@
+  networks: NetworkType[],
+  pageKeys: Partial<Record<NetworkType, string>> = {},
+) => {
+  const url = new URL('http://localhost:3030/api/trpc/activities.page')
```
https://github.com/status-im/status-web/pull/722/commits/eaecf05f29cee56eda56d48973de9995b95b9171

jkbktl commented on .changeset/curvy-camels-flash.md:
https://github.com/status-im/status-web/pull/722/commits/2686e68123d700b77c76438ecb2e2b6e50e6478c

jkbktl commented on .changeset/curvy-camels-flash.md:
https://github.com/status-im/status-web/pull/722/commits/2686e68123d700b77c76438ecb2e2b6e50e6478c
